acceptance_tests/features/steps/view_collection_exercise_details_state.py

```diff
@@ -0,0 +1,20 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 1803 collection exercise for the QBS survey has been created')
+def qbs_1803_exists(_):
+    pass
+
+
+@given('the internal user navigates to the collection exercise details page for QBS 1803')
+@when('the internal user navigates to the collection exercise details page for QBS 1803')
+def internal_user_views_2017_bres_collection_exercise(_):
```

Comment:
This is already being defined in view_collection_exercise_details.py can get rid of it.